docs/bibliography/diffusion-mri.md

```diff
@@ -16,6 +16,11 @@
 
 > Huber E, Henriques RN, Owen JP, Rokem A, Yeatman JD (2019) Applying microstructural models to understand the role of white matter in cognitive development. Dev Cogn Neurosci 36(February):100624.
 
+**There are so many tractography algorithms. Which one is the best?**
+There is no simple answer to this question: it depends on your goals, the data acquisition as well as the subjects being studied. For example, many of the cutting-edge algorithms depend on collecting many diffusion directions across multiple b-values and this often isn't feasible in clinical studies or pediatric studies. Understanding the tradeoffs between different algorithms is important. Recent work by Girarand and colleagues (2020) compared the compendium of tractography algorithms implemented in mrtrix and DIPY to "gold-standard" tracer studies done in resus macaque monkey. This recent paper provides a thoughtful description of how to think about gold standards and how to evaluate the tradeoffs between different algorithms. They conclude that many tractography algorithms are quite good in terms of sensitivity/specificity and provide useful guidance in distinguishing between different algorithms.
```

Comment:
```suggestion
There is no simple answer to this question: it depends on your goals, the data acquisition as well as the subjects being studied. For example, many of the cutting-edge algorithms depend on collecting many diffusion directions across multiple b-values and this often isn't feasible in clinical studies or pediatric studies. Understanding the tradeoffs between different algorithms is important. Recent work by Girard and colleagues [@girard2020cortical] compared the compendium of tractography algorithms implemented in mrtrix and DIPY to "gold-standard" tracer studies done in rhesus macaque monkey. This recent paper provides a thoughtful description of how to think about gold standards and how to evaluate the tradeoffs between different algorithms. They conclude that many tractography algorithms are quite good in terms of sensitivity/specificity and provide useful guidance in distinguishing between different algorithms.
```

Reply:
Very nice! 

The syntax for markdown citations from the bibliography is `[@girard2020cortical]` (or whatever the key is for the reference you'd like to cite).